client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -161,6 +167,14 @@ class Upload extends React.Component {
 		const file = files[ 0 ] || files.item( 0 );
 		debug( 'zip file:', file );
 
+		if ( file.size > MAX_UPLOADED_THEME_SIZE ) {
+			notices.error(
+				translate( 'Theme zip is too big. Max theme size is 50 MB.' )
```

Comment:
Maybe `Theme zip is too large. Please upload a theme under 50 MB.`

Also it appears that there's a too large string above that says it should be 10MB. Is that a different case or should it be updated to 50MB?